cogs/commands/summarise.py

```diff
@@ -31,20 +33,30 @@ def __init__(self, bot: Bot):
         openai.api_key = CONFIG.OPENAI_API_KEY
         self.system_prompt = "People yap too much, I don't want to read all of it. In 200 words or less give me the gist of what is being said. Note that the messages are in reverse chronological order:"
 
+    def build_prompt(self, response_word_limit, bullet_points, channel_name):
+
+        response_word_limit = 500 if response_word_limit > 500 else response_word_limit
+        bullet_points = "Put it in bullet points for readability." if bullet_points else ""
+        prompt = f"""People yap too much, I don't want to read all of it. The topic is something related to {channel_name}. In {response_word_limit} words or less give me the gist of what is being said. {bullet_points} Note that the messages are in reverse chronological order:
```

Comment:
imo its probably gonna be easier to say in 2 sentances or less to cut down on response size

Reply:
in the future i will make the response size dynamic based on the number of words it scrapes, but ill  keep it under two sentences for now